appcues/src/main/java/com/appcues/analytics/AnalyticsTracker.kt

```diff
@@ -19,16 +15,9 @@ internal class AnalyticsTracker(
     private val activityBuilder: ActivityRequestBuilder,
     private val experienceLifecycleTracker: ExperienceLifecycleTracker,
     private val analyticsPolicy: AnalyticsPolicy,
+    private val analyticsQueueManager: AnalyticsQueueManager,
```

Comment:
a lot of simplification in this class now that we have an extra AnalyticsQueueManager to handle all queue functions